integ/test_init/test-guide.md

```diff
@@ -0,0 +1,107 @@
+###  Integration Test for Init Process of Fluent Bit on ECS
+
+Following this instruction, you will take the integration test for init process and get the test result
+
+
+
+#### Requirements
+
+1. [install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+2. [set up AWS account credential](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html)
+3. [install AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html)
+
+
+
+#### Deploy test resources
+
+* Create some Fluent Bit config files locally  (you can use any config files that are valid).
```

Comment:
Can you give some configs in the test files, that will make it easy to run. The steps should preferably be just commands to run as much as possible.

Reply:
Added~